packages/cli/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "display": "Node",
   "compilerOptions": {
     "strict": true,
-    "target": "es6",
+    "target": "es2020",
```

Comment:
Needed to bump this since "import.meta" is not available in es6 (see: https://github.com/evanw/esbuild/blob/8c71947edbe5a158fec3a6d1cbfea1e8d5cdee70/CHANGELOG-2020.md?plain=1#L2698)

According to https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping, es2020 has support back to Node 14. Since we're requiring at least Node 20, this change should be fine.